### PR TITLE
chore: update new storybook module info

### PR DESCRIPTION
**Pull request by chakAs3**
Storybook Module is completely overwrites the old, so i have updated relevant info

### CODE REVIEW

**Comment by chakAs3** [comment]
@antfu would you merge this PR, i updated storybook module with correct information.

File: modules/storybook.yml [hunk 1]
```diff
@@ -7,18 +7,16 @@ npm: '@nuxtjs/storybook'
...
-    twitter: a_birang
-  - name: Pooya Parsa
-    github: pi0
-    twitter: _pi0_
+  - name: Chakir QATAB
+    github: chakAs3
+    twitter: ChakirQatab
 compatibility:
-  nuxt: ^2.0.0 || ^3.0.0
+  nuxt: ^3.6.5
```
Comment (danielroe) [inline modules/storybook.yml:18]: As we still have a module that works with Nuxt 2:

```suggestion
  nuxt: ^2.0.0 || ^3.6.5
```

File: modules/storybook.yml [hunk 2]
```diff
@@ -7,18 +7,16 @@ npm: '@nuxtjs/storybook'
...
-    twitter: a_birang
-  - name: Pooya Parsa
-    github: pi0
-    twitter: _pi0_
+  - name: Chakir QATAB
+    github: chakAs3
+    twitter: ChakirQatab
 compatibility:
-  nuxt: ^2.0.0 || ^3.0.0
+  nuxt: ^2.0.0 || ^3.6.5
```
Comment (pi0) [inline modules/storybook.yml:18]: Would you please add instructions for Nuxt 2 support status in the main `README`? 🙏🏼

**Review by pi0 — APPROVED** [review]

**Review by chakAs3 — COMMENTED** [review]
@pi0 do you need nuxt 2 instruction before merging it ?

**Comment by pi0** [comment]
It would be nice since we are marking compatibility as `2 || 3` and someone can try on Nuxt 2/bridge which fails. (We can rename old module to `storybook-legacy` as slug or something. /cc @farnabaz)

**Comment by chakAs3** [comment]
> It would be nice since we are marking compatibility as `2 || 3` and someone can try on Nuxt 2/bridge which fails. (We can rename old module to `storybook-legacy` as slug or something. /cc @farnabaz)

ok good idea, i will do my tests, first then see what we ca do thanks 👍

**Review by pi0 — APPROVED** [review]
would be nice to add compatibility.versionMap in the future to unbreak nuxt 2 cli installs

**Pull request merged**